src/pages/settings/PasswordPage.js

```diff
@@ -133,7 +133,7 @@ class PasswordPage extends Component {
         if (!this.validate()) {
             return;
         }
-        User.changePasswordAndNavigate(this.state.currentPassword, this.state.newPassword);
```

Comment:
So, this never really navigated?

Reply:
Apparently not 🤷🏽

Reply:
Yeah, seems like the original function never had any navigation in it 😄